exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServerBuilder.java

```diff
@@ -21,6 +20,7 @@ public final class PrometheusHttpServerBuilder {
 
   private String host = DEFAULT_HOST;
   private int port = DEFAULT_PORT;
+  private PrometheusRegistry prometheusRegistry = PrometheusRegistry.defaultRegistry;
```

Comment:
This means that by default, PrometheusHttpServer will serve all metrics collected from `MetricCollector` instances registered with the default prometheus registry, rather than only metrics from SdkMeterProvider?

If so, I don't think that should be the default behavior. I think the default registry should be `new PrometheusRegistry`. 

I'm not sure its a good idea to allow users to set their own registry, since doing so could cause conflicts which `SdkMeterProvider` is not responsible for. @fstab what happens if multiple `MetricCollectors` are registered with the default registry and return conflicting MetricSnapshots?

Reply:
I changed the default to `new PrometheusRegistry()`.

However, I think it would be good to allow users to set their own registry, because that enables an easy transition path between the Prometheus library and the OTel SDK. You can simply use both libraries and register all metrics with the same Prometheus registry.